staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6215,6 +6215,13 @@ type WindowsSecurityContextOptions struct {
 	// PodSecurityContext, the value specified in SecurityContext takes precedence.
 	// +optional
 	RunAsUserName *string `json:"runAsUserName,omitempty" protobuf:"bytes,3,opt,name=runAsUserName"`
+
+	// HostProcess determines if a container should be run as a 'Host Process' container.
+	// This field is alpha-level and will only be honored by components that enable the
+	// WindowsHostProcessContainers feature flag. Setting this field without the feature
+	// flag will result errors when validating the Pod.
```

Comment:
```suggestion
	// flag will result in errors when validating the Pod.
```

finally commented on the source of it =)